healthcare/dicomweb_instance_retrieve.go

```diff
@@ -46,18 +52,18 @@ func dicomWebRetrieveInstance(w io.Writer, projectID, location, datasetID, dicom
 
 	defer resp.Body.Close()
 
-	respBytes, err := ioutil.ReadAll(resp.Body)
-	if err != nil {
-		return fmt.Errorf("could not read response: %v", err)
-	}
-
-        if resp.StatusCode > 299 {
-                return fmt.Errorf("RetrieveInstance: status %d %s: %s", resp.StatusCode, resp.Status, respBytes)
+	if resp.StatusCode > 299 {
+                return fmt.Errorf("RetrieveInstance: status %d %s: %s", resp.StatusCode, resp.Status, resp.Body)
         }
 
-	if err = ioutil.WriteFile(outputFile, respBytes, 0644); err != nil {
-		return err
-	}
+	file, err := os.Create(outputFile)
+        if err != nil {
```

Comment:
Need to run `gofmt`. Your editor should be able to do it on save.

Reply:
Done for all files.